app/src/main/java/net/gsantner/markor/ui/hleditor/Highlighter.java

```diff
@@ -15,7 +15,7 @@
 import android.graphics.Typeface;
 import android.support.annotation.ColorInt;
 import android.support.annotation.NonNull;
-import android.text.Editable;
+import android.text.Spannable;
```

Comment:
Most of these functions can be static and only require a Spannable (not an Editable). This change makes the Highlighter class more flexible.